testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/DemoServletsAdapterTest.java

```diff
@@ -826,6 +825,9 @@ public void testTokenConcurrentRefresh() {
         } finally {
             executor.shutdownNow();
         }
+
+        // Revert times
+        setAdapterAndServerTimeOffset(0, tokenRefreshPage.toString());
```

Comment:
Doesn't the adapter reset the time automatically? If memory serves me correctly that's done on the server-side.

Reply:
It does for the auth server, but not for the adapter container because there needs to be a Filter deployed and Before/After methods are not aware of the URL on which Filter operates.

Reply:
Thanks for the explanation. LGTM then. I'll let @hmlnarik merge this one though, since I clearly lack knowledge on the adapter testing side ;)